server/gitrest/packages/gitrest-base/src/utils/repositoryManagerFactoryBase.ts

```diff
@@ -117,7 +117,10 @@ export abstract class RepositoryManagerFactoryBase<TRepo> implements IRepository
             params.storageRoutingId.tenantId,
             params.storageRoutingId.documentId,
             this.storageDirectoryConfig.useRepoOwner ? params.repoOwner : undefined);
-        const directoryPath = helpers.getGitDirectory(repoPath, this.storageDirectoryConfig.baseDir);
+        const directoryPath = helpers.getGitDirectory(
+            repoPath,
```

Comment:
nit: one thing I found confusing when parsing through this change and the related code, was `repoPath` is now only a small piece of the actual filesystem path to the repo. Perhaps a better name for this variable (and `getRepoPath`) would be `repoName`?

Also totally not required, feel free to merge if I'm being pedantic

Reply:
For some reason the permalink I used did not show up in my reply above... This is the other `repoName` I was referring to:
https://github.com/microsoft/FluidFramework/blob/0b3d1e849a72e644edc905fd5fb601649c8caa7c/server/gitrest/packages/gitrest-base/src/utils/repositoryManagerFactoryBase.ts#L141-L144